### ISSUE TITLE
Unable to resolve module `AccessibilityInfo`

### ISSUE THREAD

**Issue by jseaidou** [facebook/react-native#26366]
I am aware of previous issues like #14209, but these happened a couple years ago and the resolution for most of them is to change versions that are too old now. 

I ran yarn install on my project, and then `react-native start -- --reset-cache`. The output of creating the bundle is: 

```
error: bundling failed: Error: Unable to resolve module `AccessibilityInfo` from `/Users/jamil/code/myproject/node_modules/react-native/Libraries/react-native/react-native-implementation.js`: Module `AccessibilityInfo` does not exist in the Haste module map
```

React Native version:
```
System:
    OS: macOS 10.14.6
    CPU: (8) x64 Intel(R) Core(TM) i7-8559U CPU @ 2.70GHz
    Memory: 2.79 GB / 16.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 12.9.1 - /usr/local/bin/node
    Yarn: 1.17.3 - /usr/local/bin/yarn
    npm: 6.10.3 - /usr/local/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    iOS SDK:
      Platforms: iOS 12.4, macOS 10.14, tvOS 12.4, watchOS 5.3
  IDEs:
    Xcode: 10.3/10G8 - /usr/bin/xcodebuild
  npmPackages:
    react: ^16.9.0 => 16.8.6
    react-native: ^0.60.5 => 0.60.5
  npmGlobalPackages:
    react-native-cli: 2.0.1
```
## Steps To Reproduce

1. `yarn install` with dependencies:
```
{
    "dependencies": {
    "@react-native-community/async-storage": "^1.6.1",
    "@react-native-community/geolocation": "^2.0.2",
    "@react-native-community/netinfo": "^4.1.5",
    "lodash": "latest",
    "moment": "^2.23.0",
    "prop-types": "^15.7.2",
    "react": "^16.9.0",
    "react-native": "^0.60.5",
    "react-native-animatable": "^1.3.2",
    "react-native-background-upload": "^5.1.0",
    "react-native-camera": "^3.2.0",
    "react-native-circular-progress": "^1.3.0",
    "react-native-contacts": "^5.0.2",
    "react-native-customized-image-picker": "^0.1.4",
    "react-native-datepicker": "^1.7.2",
    "react-native-device-info": "^2.3.2",
    "react-native-easy-toast": "^1.2.0",
    "react-native-elements": "^1.1.0",
    "react-native-fbsdk": "git+https://github.com/facebook/react-native-fbsdk.git",
    "react-native-firebase": "^5.5.6",
    "react-native-gesture-handler": "^1.4.0",
    "react-native-image-crop-picker": "^0.25.0",
    "react-native-image-picker": "^1.0.2",
    "react-native-keyboard-aware-scroll-view": "^0.9.1",
    "react-native-linear-gradient": "^2.5.4",
    "react-native-modal": "^11.3.1",
    "react-native-parsed-text": "^0.0.21",
    "react-native-permissions": "git+https://github.com/BaderSerhan/react-native-permissions",
    "react-native-phone-input": "^0.2.4",
    "react-native-progress": "^3.5.0",
    "react-native-reanimated": "^1.2.0",
    "react-native-share": "^2.0.0",
    "react-native-size-matters": "^0.2.1",
    "react-native-sms": "^1.9.0",
    "react-native-svg": "^9.6.4",
    "react-native-text-ticker": "^0.18.0",
    "react-native-vector-icons": "^6.6.0",
    "react-native-video": "^5.0.0",
    "react-native-video-processing": "git+https://git@github.com/shahen94/react-native-video-processing.git",
    "rn-fetch-blob": "^0.10.16",
    "yarn": "^1.17.3"
  },
  "devDependencies": {
    "babel-jest": "23.6.0",
    "babel-plugin-transform-remove-console": "^6.9.4",
    "jest": "23.6.0",
    "react-test-renderer": "16.6.3"
  },
  "jest": {
    "preset": "react-native"
  }
}
 ```

2. `react-native start -- --reset-cache`

3. `curl http://localhost:8081/index.bundle -o main.jsbundle`

**Comment by teeolendo** [comment]
Unable to reproduce the error unfortunately. Mind running `react-native info` to get your full environment info?

**Comment by jseaidou** [comment]
@teeolendo the `react native info` is in the description, under`react native version`

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
